optimus/src/main/java/co/mobiwise/optimus/Optimus.java

```diff
@@ -155,7 +158,6 @@ public static void putStringSet(String key, HashSet<String> value) {
   public static Editor removeAll() {
     final Editor editor = getOptimusPref().edit().clear();
     editor.apply();
-    return editor;
```

Comment:
return durmasının bence sakıncası yok.

Reply:
Bana göre (belki bir nedeni vardır) ihtiyacı olmadığı için kaldırdım. Çünkü removeAll metodunun  görevi silmek, editor'u farklı bir metodla almalıyız.